### ISSUE TITLE
lodash@4.17.11 is vulnerable

### ISSUE THREAD

**Issue by AndreyBelym** [inikulin/publish-please#91]
`lodash` versions lesser or equal to `4.17.11` have a security vulnerability: https://github.com/lodash/lodash/issues/4348. `npm` started reporting this problem in `publish-please` today:
```
found 2 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
 
ERRORS
  * Vulnerability found in publish-please -> lodash
  * Vulnerability found in publish-please -> inquirer -> lodash
```

**Comment by hdorgeval** [comment]
Hi @AndreyBelym : all my apologizes because I completely skipped your issue. Should be fixed in 5.5.1. 

Do not hesitate to ping me multiple times when I do not answer to your future issues.

**Comment by AndreyBelym** [comment]
Don't worry, everything is OK. Thank you for a good job 👍
